web/source/osk/oskManager.ts

```diff
@@ -1170,17 +993,17 @@ namespace com.keyman.osk {
 
         // Fix or release user resizing
         if('nosize' in p) {
-          if(this.resizeIcon) {
-            this.resizeIcon.style.display=(p['nosize'] ? 'none' : 'block');
+          if(this.desktopLayout) {
+            this.desktopLayout.resizeBar.allowResizing(!p['nosize']);
           }
         }
 
       }
       // Fix or release user dragging
       if('nomove' in p) {
         this.noDrag=p['nomove'];
-        if(this.pinImg) {
-          this.pinImg.style.display=(p['nomove'] || !this.userPositioned) ? 'none' : 'block';
+        if(this.desktopLayout) {
+          this.desktopLayout.titleBar.showPin(!(p['nomove'] || !this.userPositioned));
```

Comment:
This test is overly negative 😁 

```suggestion
          this.desktopLayout.titleBar.showPin(!p['nomove'] && this.userPositioned);
```